edx/analytics/tasks/student_properties_per_ora_tags_dist.py

```diff
@@ -90,11 +90,12 @@ def mapper(self, line):
         student_properties = event.get('context').get('asides', {}).get('student_properties_aside', {})\
             .get('student_properties', {})
 
-        overload_items = {'course': course, 'term': run}
+        overload_items = {'course': course, 'term': run, 'courses': course}
```

Comment:
Do we need the same for terms, then?

Reply:
actually not, I've removed it.

Reply:
We've added transformation `terms`, `runs` and `run` to `term`